_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -0,0 +1,164 @@
+       cheaper is used; if they both result in the same cost, partial
+       spend avoidance is used.
+
+    3. `maxapsfee` set to greater than `0`: partial spend avoidance is
+       used whenever the maximum additional cost it adds to the
+       transaction is the passed amount.  For example,
+       `-maxapsfee=0.00001000` means the wallet will avoid partial
+       spends if the absolute fee difference is up to 1,000 sats.
+
+- [Bitcoin Core #19550][] adds a new `getindexinfo` RPC that lists each
+  optional index that has been enabled, how many blocks have been
+  indexed so far, and whether those blocks constitute syncing all the way
+  to the tip of the block chain.
+
+- [C-Lightning #3954][] updates both the `fundpsbt` and `utxopsbt` RPCs
+  so that they can each take a `locktime` parameter that specifies the
+  nLockTime of the transaction to create.  The PR notes that this is
+  "required for [dual funding][bolts #524], where the opener sets [the
+  locktime]".
+
+- [BIPs #955][] updates [BIP174][] to standardize supplying hash preimages in
+  [PSBT][topic psbt] input records. The standardization of these preimage fields
+  was [found to be necessary for miniscript-aware finalizers][psbt ext], though
+  they can be used by any PSBT finalizer needing to satisfy hash preimage
+  challenges (e.g. for onchain LN commitment transactions).
+
+- [BOLTs #688][] adds support for [anchor outputs][topic anchor outputs]
+  to the LN specification.  This extends commitment transactions with
+  two extra outputs---one for each counterparty---which can be used for
+  Child Pays For Parent ([CPFP][topic cpfp]) fee bumping.   With this
+  change, it becomes possible to fee bump commitment transactions that
+  may have been signed days or weeks earlier, when the current feerates
+  would've been hard to predict.  In practice, LN nodes using anchor
+  outputs should normally pay lower fees because there's no longer any
+  incentive to overestimate fees. Anchor outputs also provide greater security
+  because, if feerates do increase beyond what was predicted, the node
+  can fee bump its commitment transaction.  Various degrees of support
+  for anchor outputs have already been merged into several LN
+  implementations.
+
+- [BOLTS #785][] updates the minimum CLTV expiry delta to 18 blocks.  To
+  ensure that the latest state gets recorded onchain, a channel should
+  be unilaterally closed when there are only this many blocks until an
+  LN payment has to be settled.  However, the higher the expiry is, the
+  more time a payment could become temporarily stuck in an open channel
+  (either by accident or deliberately).  This has led some LN
+  implementations to use route-finding algorithms that optimize for
+  routes with low CLTV expiry deltas, which has in turn led some users
+  to set their deltas to values that are especially unsafe.  This new
+  minimum should help prevent inexperienced users from naively setting
+  an unsafe value.
```

Comment:
"unsafe" mentioned twice without saying why it is unsafe. A risk associated with higher expiry values is described above; the same for low values may be good here

Reply:
I think it is explained, "To ensure that the latest state is recorded onchain [before] an LN payment has to be settled."